cpp/tests/strings/split_tests.cpp

```diff
@@ -275,77 +275,44 @@ TEST_F(StringsSplitTest, AllNullsCase)
   EXPECT_TRUE(column.null_count() == column.size());
 }
 
-TEST_F(StringsSplitTest, ContiguousSplitRecord)
+TEST_F(StringsSplitTest, SplitRecord)
 {
   std::vector<const char*> h_strings{" Héllo thesé", nullptr, "are some  ", "tést String", ""};
   cudf::test::strings_column_wrapper strings(
     h_strings.begin(),
     h_strings.end(),
     thrust::make_transform_iterator(h_strings.begin(), [](auto str) { return str != nullptr; }));
 
-  cudf::strings_column_view strings_view(strings);
-
-  std::vector<const char*> h_expected1{"", "Héllo", "thesé"};
-  cudf::test::strings_column_wrapper expected1(h_expected1.begin(), h_expected1.end());
-  std::vector<const char*> h_expected2{};
-  cudf::test::strings_column_wrapper expected2(h_expected2.begin(), h_expected2.end());
-  std::vector<const char*> h_expected3{"are", "some", "", ""};
-  cudf::test::strings_column_wrapper expected3(h_expected3.begin(), h_expected3.end());
-  std::vector<const char*> h_expected4{"tést", "String"};
-  cudf::test::strings_column_wrapper expected4(h_expected4.begin(), h_expected4.end());
-  std::vector<const char*> h_expected5{""};
-  cudf::test::strings_column_wrapper expected5(h_expected5.begin(), h_expected5.end());
-
-  std::vector<std::unique_ptr<cudf::column>> expected_columns;
-  expected_columns.push_back(expected1.release());
-  expected_columns.push_back(expected2.release());
-  expected_columns.push_back(expected3.release());
-  expected_columns.push_back(expected4.release());
-  expected_columns.push_back(expected5.release());
-
-  auto result = cudf::strings::contiguous_split_record(strings_view, cudf::string_scalar(" "));
-  EXPECT_TRUE(result.column_views.size() == expected_columns.size());
-  for (size_t i = 0; i < result.column_views.size(); ++i) {
-    cudf::test::expect_columns_equal(result.column_views[i], *expected_columns[i]);
-  }
+  auto result =
+    cudf::strings::split_record(cudf::strings_column_view(strings), cudf::string_scalar(" "));
+  cudf::lists_column_view lcv(result->view());
+  cudf::test::strings_column_wrapper expected(
+    {"", "Héllo", "thesé", "are", "some", "", "", "tést", "String", ""});
+  cudf::test::fixed_width_column_wrapper<int32_t> offsets({0, 3, 3, 7, 9, 10});
```

Comment:
I thought that @nvdbaranec had APIs to create list columns for tests that would make this even smaller.  Do those not work for string columns?

https://github.com/rapidsai/cudf/blob/ce826c57c8cafb7d78f33873ca8c0c030fa42087/cpp/tests/utilities/column_wrapper.hpp#L587-L606